04_documentation/technical_writing_slides.md

```diff
@@ -97,9 +106,9 @@ Much content of this lecture is taken from [Write the Docs](https://www.writethe
 ## Technical Writing in Research Software
 
 - Like in a startup, **no dedicated technical writer in team** (even for large projects). Everybody does everything.
-    - -> You need writing skills.
-- In research, we anyway write a lot (papers, proposals, lecture material, ...).
-    - -> Writing/communication skills are very important anyway.
+    - --> You need writing skills.
+- In research, we write a lot (papers, proposals, lecture material, ...).
```

Comment:
```suggestion
- In research, one writes a lot (papers, proposals, lecture material, ...).
```
I think we can avoid *we* where is may sound like *we the researchers, and you the students* as a differentiation.